### PR TITLE
Improve readability of .cgtest files with better separators

### PR DESCRIPTION
**Pull request by piotrswigon**
Plus some minor improvements:
- comment on the utility of unused variable.
- pass input file + test name as the filename for manifest parsing, to improve debuggability.

### CODE REVIEW

File: src/tools/tests/goldens/kotlin-connection-type-generator.cgtest [hunk 1]
```diff
@@ -1,60 +1,60 @@
-[header]
+-----[header]-------------------------------------------------------------------
```
Comment (bgogul) [inline src/tools/tests/goldens/kotlin-connection-type-generator.cgtest:1]: Just curious. Is this expected to be 80 characters? If the intention is to visually separte the sections better, something shorter like `----[header]----` or `#### [header] ####`might suffice?

Comment (piotrswigon) [reply src/tools/tests/goldens/kotlin-connection-type-generator.cgtest:1]: Visual separation was the intention. I can change to ----[header]---- if you prefer that.

Comment (piotrswigon) [reply src/tools/tests/goldens/kotlin-connection-type-generator.cgtest]: Done for you enjoyment :)

Comment (bgogul) [reply src/tools/tests/goldens/kotlin-connection-type-generator.cgtest]: Thanks! :)

**Pull request merged**